pkg/client/imagestreams.go

```diff
@@ -109,7 +110,7 @@ func (c *imageStreams) UpdateStatus(stream *imageapi.ImageStream) (result *image
 // will be returned if no actual import was requested (the to fields were not set), or an ImageStream if import was requested.
 func (c *imageStreams) Import(isi *imageapi.ImageStreamImport) (*imageapi.ImageStreamImport, error) {
 	result := &imageapi.ImageStreamImport{}
-	if err := c.r.Post().Namespace(c.ns).Resource("imageStreamImports").Body(isi).Do().Into(result); err != nil {
+	if err := c.r.Post().Namespace(c.ns).Resource("imageStreamImports").Timeout(2 * time.Minute).Body(isi).Do().Into(result); err != nil {
```

Comment:
Isn't request timeout already set into the value we send to the server?  If not, why not?